enhancements/node-pull-credentials/pull-credentials.md

```diff
@@ -0,0 +1,193 @@
+---
+title: node-artifacts-during-build-and-image-stream-import
+authors:
+  - "@rmarasch"
+reviewers:
+  - "@dmage"
+  - "@bparees"
+  - "@adambkaplan"
+approvers:
+  - "@dmage"
+  - "@bparees"
+  - "@adambkaplan"
+creation-date: 2019-12-02
+last-updated: 2019-12-10
+status: provisional
+---
+
+# Using node pull credentials during build and image stream import
+
+## Release Signoff Checklist
+
+- [ ] Enhancement is `implementable`
+- [ ] Design details are appropriately documented from clear requirements
+- [ ] Test plan is defined
+- [ ] Graduation criteria for dev preview, tech preview, GA
+- [ ] User-facing documentation is created in [openshift-docs](https://github.com/openshift/openshift-docs/)
+
+## Summary
+
+Allow OpenShift users to import and use images from any registry configured
+during or after the cluster installation by sharing node's pull credentials
+with `openshift-api` and `builder` pods.
+
+## Motivation
+
+To have image stream and builds working closely to node avoids complexity and
+redundancy as each component won't need to mount different Secrets to have
+access to information that is already present on node's filesystem.
+
+In order to provide a seamless developer experience OpenShift should allow
+users to import or to use during builds any registry credentials provided
+during or after the installation, among the registries we would like to allow
+users to interact with is `registry.redhat.io`. 
+
+Nowadays pull credentials provided during cluster install are available on
+node's filesystem. If user attempts to import an image stream from these
+locations OpenShift fails as neither `openshift-api` nor `builder` use the
+credentials provided during the installation.
+
+### Goals
+
+- Allow users to import image streams from images hosted in any registry
+  provided during or after cluster installation, including `registry.redhat.io`.
```

Comment:
Allow users to use container images from any registry provided during or after cluster installation

Reply:
fixed.